plugins/git/README.md

```diff
@@ -59,6 +59,8 @@ plugins=(... git)
 | gcpa                 | git cherry-pick --abort                                                                                                          |
 | gcpc                 | git cherry-pick --continue                                                                                                       |
 | gcs                  | git commit -S                                                                                                                    |
+| gcss                 | git commit -S -s                                                                                                                 |
+| gcssm                | git commit -S -s -m                                                                                                              |
```

Comment:
Those were missing but they're implemented

Reply:
Implemented in https://github.com/ohmyzsh/ohmyzsh/commit/08751210e3204c4ba330bce1ff52b1906b92f850 without adding them to the README